jsapp/js/components/permissions/userPermissionRow.component.tsx

```diff
@@ -12,6 +12,7 @@ import {getPermLabel, getFriendlyPermName} from './utils';
 
 interface UserPermissionRowProps {
   assetUid: string;
+  displayControls: boolean;
```

Comment:
In most places we tend to use names that are "facts" rather than "orders" (don't know how to explain this better, sorry 😅). So instead of having that property `displayControls` that requires parent component to know things about `UserPermissionRow`, we could pass the same info, but let the `UserPermissionRow` decide for itself how to use it - so `useCanEditPerms` would be better :)